pkgs/top-level/all-packages.nix

```diff
@@ -1115,6 +1115,8 @@ with pkgs;
 
   hid-listen = callPackage ../tools/misc/hid-listen { };
 
+  home-manager = callPackage ../applications/misc/home-manager {};
```

Comment:
I think Home Manager would be [considered](https://nixos.org/nixpkgs/manual/#sec-hierarchy) a tool rather than an application since it is not intended for interactive use.